### PR TITLE
Syndicate borg update, add medical syndicate borg, energy sword refactor

### PR DESCRIPTION
**Pull request by Markolie**
Changes:
1) Refactors energy sword code, so we're up-to-date with /tg/. Also tweaks some of the numbers.
2) Adds medical syndicate cyborgs, which can be chosen as an alternative to the assault cyborg: https://github.com/tgstation/-tg-station/pull/11809 by Xhuis.
3) Adds a surgery table and a full set of surgery equipment to the syndicate shuttle.
4) Updates twohanded code to /tg/'s latest.
5) Ensures none of the APC's/air alarms on MO19 trigger alarms.
6) Adds the syndicate dart gun to the uplink, which is a smaller and suppressed syringe gun.

The Syndicate medical cyborg comes with everything a medical cyborg has, plus a Syndicate cyborg hypospray with a variety of chems, a CO2 jetpack, a crowbar, an operative pinpointer and an energy saw (which is a circular saw that can be activated to do more damage).

### CODE REVIEW

File: code/game/objects/items/weapons/melee/energy.dm [hunk 1]
```diff
@@ -1,55 +1,101 @@
...
 	sharp = 1
 	edge = 1
+	var/hacked = 0
+	var/blade_color
+	
+/obj/item/weapon/melee/energy/sword/New()
+	if(_color == null)
+		_color = pick("red", "blue", "green", "purple")
+
+/obj/item/weapon/melee/energy/sword/IsShield()
```
Comment (tigercat2000) [inline code/game/objects/items/weapons/melee/energy.dm]: errr if I recall correctly this proc has much more affect here than it does on -tg-

**Comment by Fox-McCloud** [comment]
Very niceeee.

File: code/modules/reagents/reagent_containers/borghydro.dm [hunk 2]
```diff
@@ -22,6 +22,14 @@
...
 /obj/item/weapon/reagent_containers/borghypo/crisis
 	reagent_ids = list("salglu_solution", "epinephrine", "sal_acid")
+	
+/obj/item/weapon/reagent_containers/borghypo/syndicate
+	name = "syndicate cyborg hypospray"
+	desc = "An experimental piece of Syndicate technology used to produce powerful restorative nanites used to very quickly restore injuries of all types. Also metabolizes potassium iodide, for radiation poisoning, and morphine, for offense."
+	icon_state = "borghypo_s"
+	charge_cost = 20
+	recharge_time = 2
+	reagent_ids = list("omnizine", "salbutamol", "mannitol", "mutadone", "charcoal", "potass_iodide")
```
Comment (Fox-McCloud) [inline code/modules/reagents/reagent_containers/borghydro.dm]: this should be reagent_ids = list("syndicate_nanites", "potass_iodide", "ether")

**Comment by Fox-McCloud** [comment]
Oh, what aobut the defib changes and the syndicate syringe gun?

**Comment by FalseIncarnate** [comment]
:+1: especially for making MO19 alerts not show up on station consoles. Not a massive thing, but takes away some of the excitement of gateway exploring when Atmos Techs and Engineers can tell you what mission it is without going through the gateway.

File: code/modules/reagents/reagent_containers/borghydro.dm (hunk 2 above)
Comment (Markolie) [inline code/modules/reagents/reagent_containers/borghydro.dm]: I don't think the borgs on /tg/ have the nanites. I'll have to check.

**Comment by Markolie** [comment]
We still have the borg defib, so there's no need to make any defib changes. And the syringe gun isn't part of the borgs loadout.

**Pull request merged**